### ISSUE TITLE
[Medium] Chrome+JAWS: Status is not announced by Screen Reader

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9452]
## Action Performed:
1. Open Expensify account on Web.
2. Select any chat room in your list.
3. Tap the NEW (green plus) button on the bottom left of the screen.
4. Open the folder with the desired file on your computer.
5. Drag a photo from the folder to the chat room.
6. Notice that opened menu closed and user can drop file to the chat.
7. Observe the status of file sent is not announced by Screen Reader.

## Expected Result:
File sent/uploaded information should be announced by Screen Reader

## Actual Result:
File sent/uploaded information is not announced by Screen Reader

## Workaround:
Yes

## Area issue was found in:
Chat Section

## Failed WCAG checkpoints
4.1.3

## User impact:
When status messages are not programmatically indicated, users of assistive technology, such as screen readers, may be unaware of the status change.

## Suggested resolution:
Assign appropriate roles or properties to status messages so that the new content is spoken by screen readers.
The aria-live property can be used to notify assistive technology when messages are injected into a live region container. Special case live region roles, such as role="alert", role="status", role="progressbar", role="log" etc., can also be used instead of applying live region properties directly. Developers should avoid the use of role="alert" or aria-live="assertive" as it may interrupt the user's text-to-speech.

Refer: 
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22.html

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

Version Number:*v1.1.76-2*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

https://user-images.githubusercontent.com/98817464/173617398-b02a57cc-a330-4e06-9d02-cc62a92503c7.mp4
